src/cliargs/core.lua

```diff
@@ -149,7 +149,8 @@ local function create_core()
   ---        The number of columns assigned to the argument descriptions, set to
   ---        0 to auto set the total width to 72.
   function cli:set_colsz(key_cols, desc_cols)
-    colsz = { key_cols or colsz[1], desc_cols or colsz[2] }
+    colsz[1] = key_cols or colsz[1]
```

Comment:
Because the current implementation is surprising. If the value could change over time, there should be a getter instead imo.

I'll be happy to make the change, I can do it tomorrow since it's weekend.